protocol/0012-POSR-position_resolution.md

```diff
@@ -110,15 +110,17 @@ Currently only one liquidation strategy is supported and its defined by the foll
 - `disposal time step` (min: `1s`, max: `1h`): network attempts to unload its position in a given market every time it goes out of auction and then every `disposal time step` seconds as long as market is not in auction mode and while the network's position is not equal to `0`,
 - `disposal fraction` (min: `0.01`, max: `1`): fraction of network's current open volume that it will try to reduce in a single disposal attempt,
 - `full disposal size` (min: `0`, max: `max int`): once net absolute value of network's open volume is at or below that value, the network will attempt to dispose the remaining amount in one go,
-- `max fraction of book side within liquidity bounds consumed` (min: `0`, max: `1`): once the network chooses the size of its order (`s_candidate`) the effective size will be calculated as `s_effective=min(m*N, s_candidate)`, where `N` is the sum of volume (on the side of the book with which the network's order will be matching) that falls within the range implied by the `market.liquidity.priceRange` [parameter](./0044-LIME-lp_mechanics.md#market-parameters) and `m` is the `max fraction of book side within liquidity bounds consumed`.
+- `disposal slippage range` (decimal `>0` with default of `0.1` which is interpreted as `10%`, existing markets to default to their current SLA range). Just like [SLA range](./0044-LIME-lp_mechanics.md) these are taken from `mid_price` during continuous trading or indicative uncrossing price during auctions so the lower bounds becomes `max(0,price x (1-range)` and upper bound `price x (1+range)`. 
+- `max fraction of book side within bounds consumed` (min: `0`, max: `1`): once the network chooses the size of its order (`s_candidate`) the effective size will be calculated as `s_effective=min(m*N, s_candidate)`, where `N` is the sum of volume (on the side of the book with which the network's order will be matching) that falls within the range implied by the `disposal slippage range` and `m` is the `max fraction of book side within liquidity bounds consumed`.
 
-Assume the price range implied by the `market.liquidity.priceRange` is `[a, b]`. Once the network has worked out a size of its immediate or cancel limit order it sets it's price to `a` if it's a sell order or `b` if it's a buy order, and it submits the order.
+Assume the price range implied by the `disposal slippage range` is `[a, b]`. Once the network has worked out a size of its immediate or cancel limit order it sets its price to `a` if it's a sell order or `b` if it's a buy order, and it submits the order.
 
 Note that setting:
 
 - `disposal time step` = `0s`,
 - `disposal fraction` = `1`,
 - `full disposal size` = `max int`,
+- `disposal slippage range` = `10`,
```

Comment:
> `disposal slippage range` (decimal `>0` with default of `0.1` which is interpreted as `10%`,

If slippage range is 10, doesn't that mean 1000%? Would have to double-check if that makes sense

Reply:
Indeed.

Reply:
I am also ok with `1`; anyway it's just an example and for migration we explicitly say "take the SLA range".

Reply:
I do think it's worth considering values > 1 here. We should handle cases where we end up with a price of `mid_price - 10 * mid_price` doesn't "overflow"/ignore the sign and results in an order of `9 * mid price`, but instead we should treat it as a de-facto market order